packages/venia-ui/lib/components/CheckoutPage/checkoutPage.js

```diff
@@ -32,18 +32,24 @@ const CheckoutPage = props => {
         // SHIPPING_ADDRESS, SHIPPING_METHOD, PAYMENT, REVIEW
         checkoutStep,
         handleSignIn,
-        isGuestCheckout,
         isCartEmpty,
+        isGuestCheckout,
         isLoading,
+        isUpdating,
         placeOrder,
         receiptData,
+        // TODO: Utilize this setter when making a mutation
+        // setIsUpdating,
         setShippingInformationDone,
         setShippingMethodDone,
         setPaymentInformationDone
     } = talonProps;
 
     const classes = mergeClasses(defaultClasses, propClasses);
 
+    const windowSize = useWindowSize();
+    const isMobile = windowSize.innerWidth <= 960;
```

Comment:
Hiding a bit of UI should be done in CSS with `display: none`, not in JS with `useWindowSize`. There are exceptions, but this isn't one of them.

Reply:
The only way out of this that I see is applying a class based on the step, and then handling the condition with css media query. Sound good?